docker/build-dapp-fs.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright 2022 Cartesi Pte. Ltd.
+# Copyright 2023 Cartesi Pte. Ltd.
```

Comment:
We removed the copyright years from the rollups repo, so we no longer have to update them. You should consider doing it here.

Reply:
Removed at 77a774c

**NOTE:** I will ammend the commit message when squashing the commits.